### DISCUSSION TITLE
Enabling markdown format images to support enlargement

### DISCUSSION THREAD

**Discussion by sunete** [mmistakes/minimal-mistakes#2595] [Q&A] [answered]
## Summary
Is it possible to enable markdown format images to support enlargement in the future?

## Motivation
I have been used this theme for a period. I offten publish posts with markdown images. But I found that images in markdown format can't be enlarged when I click it.

Markdown format:
```
![picture name](picture rul)
```
So I try to find some method to enlarge pictures in the theme guide. It gives me a solution.

Guide url: <https://mmistakes.github.io/minimal-mistakes/markup-more-images/#one-up>

I can post pictures which can be enlarged in the format below. And all the pictures in my blog are in the format below. [My blog](https://sunete.github.io/) and [blog github](https://github.com/sunete/sunete.github.io)
```
<figure> <a href="picture url"><img src="picture url"></a> </figure>
```

But it is unconvience when I insert a picture in the post. So I have a **Feature Request** that enabling markdown format images to support enlargement. Then, when I insert a picture in a post with `![ ]( )` markdown format, it could be enlarged.

![image](https://user-images.githubusercontent.com/45954867/86928245-e752a700-c166-11ea-8e17-790c8652c35f.png)

**Comment by mmistakes** [comment, marked as answer]
Theme already supports this feature... you need to link to a larger image so it has something "to enlarge".

Example:

```markdown
[![your image](image.jpg)](large-image.jpg)
```
